extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -49,13 +49,6 @@
    transition: transform 233ms cubic-bezier(0,0,.21,1);
  }
 
- .i-amphtml-toolbar {
-   z-index: 2147483646;
-   width: 100%;
-   position: fixed;
-   top: 0;
- }
-
  .i-amphtml-toolbar > ul {
```

Comment:
I left this here as it will display `toolbar-target` items horizontally when they are displayed outside of the toolbar. This is easily overridable by the developer, and its lets us take advantage of the format.